test/wolt_modal_sheet_test.dart

```diff
@@ -263,4 +263,60 @@ void main() {
     expect(tester.getTopLeft(sheetMaterial), const Offset(0.0, 504.0));
     expect(tester.getTopRight(sheetMaterial), Offset(viewSize.width, 504.0));
   }, skip: true); // [Intended]: This is skipped due to a bug in the framework.
+
+  group('WoltModalSheet page getters', () {
+    testWidgets('getters return the correct page, page index and list of pages',
```

Comment:
You added a group for "page getters" but it has a single test. Are there other page getters suitable for group?

Reply:
I group tests for fun in my free time. Kidding :) Here is the commit 81a0ac30203256d0a4a252f7145a18cedef26263